[DOCS] Check simple spelling and word order error using ChatGPT-based program (#8029)
great-expectations/great_expectations: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage.md

## 5. Confirm that the new Expectation Suites have been added

> If you have copied your existing Expectation Suites to Azure Blob Storage, run the following Python command to confirm that Great Expectations can find them:

Reviewer: ```suggestion
If you copied your existing Expectation Suites to Azure Blob Storage, run the following Python command to confirm that Great Expectations can find them:
```

## 6. Confirm that Expectations can be accessed from Azure Blob Storage

Run the following command to confirm your Expectations have been copied to Azure Blob Storage: 

```bash
great_expectations suite list
```

> If your Expectations have not been copied to Azure Blob Storage, a message indicating "No Expectations were found" is returned.

Reviewer: ```suggestion
If your Expectations have not been copied to Azure Blob Storage, the message "No Expectations were found" is returned.
```
